src/drag_face.rs

```diff
@@ -58,14 +58,14 @@ pub(crate) fn drag_face(
             ray_id: RayId::new(hit_data.camera, drag_start.pointer_id),
             face,
             initial_coord: frame.faces()[face],
-            drag_ray: Ray {
+            drag_ray: Ray3d {
                 origin: world_position,
-                direction: world_normal,
+                direction: Direction3d::new(world_normal).unwrap(),
             },
         });
     }
     for drag_end in drag_end_events.read() {
-        let Ok(mut frame) = box_frames.get_component_mut::<BoxFrame>(drag_end.target) else {
+        let Ok((mut frame, _)) = box_frames.get_mut(drag_end.target) else {
```

Comment:
Why did this change?

Reply:
`get_component_mut()` was deprecated and is removed in 0.14. I'm not familiar as why that was done on the bevy side.